bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java

```diff
@@ -419,6 +419,9 @@ public boolean addPlayer(Player p, boolean skipOwnerCheck) {
         /* used for base enter/leave event */
         isOnABase.remove(p);
         //
+        if(getServerType() != ServerType.BUNGEE && config.getConfigLoc("lobbyLoc") == null){
```

Comment:
what does this code change?

Reply:
Checks if lobbyLoc is set

Reply:
Checks if lobbyLoc is set

Reply:
y tho

Reply:
Protection against error on player exit

Reply:
but this code is getting executed on arena join

Reply:
also why dont you just set lobby location on your bw setup